content/en/docs/ambient/install/istioctl/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Install with istioctl
+description: Install Istio with support for ambient mode using the istioctl command line tool.
+weight: 10
+keywords: [istioctl,ambient]
+owner: istio/wg-environments-maintainers
+test: yes
+---
+
+{{< tip >}}
+Follow this guide to install and configure an Istio mesh with support for ambient mode.
+If you are new to Istio, and just want to try it out, follow the
+[quick start instructions](/docs/ambient/getting-started) instead.
+{{< /tip >}}
+
+This installation guide uses the [istioctl](/docs/reference/commands/istioctl/) command-line
+tool to provide rich customization of Istio installation options. It has user input validation
+to help prevent installation errors and customization options to override any aspect of the
+configuration.
+
+Using these instructions, you can select any one of Istio's built-in
+[configuration profiles](/docs/setup/additional-setup/config-profiles/)
+and then further customize the configuration for your specific needs.
+
+The `istioctl` command supports the full [`IstioOperator` API](/docs/reference/config/istio.operator.v1alpha1/)
+via command-line options for individual settings or for passing a yaml file containing an `IstioOperator`
+{{<gloss CRDs>}}custom resource (CR){{</gloss>}}.
+
+## Prerequisites
+
+Before you begin, check the following prerequisites:
+
+1. [Download the Istio release](/docs/setup/additional-setup/download-istio-release/).
+1. Perform any necessary [platform-specific setup](/docs/ambient/install/platform-prerequisites/).
+
+## Install or upgrade the Kubernetes Gateway API CRDs
+
+{{< boilerplate gateway-api-install-crds >}}
+
+## Install Istio using the ambient profile
+
+`istioctl` supports a number of [configuration profiles](/docs/setup/additional-setup/config-profiles/) that include different default options,
+and can be customized for your production needs. Support for ambient mode is included in the `ambient` profile. Install Istio with the
+following command:
```

Comment:
```suggestion
Support for ambient mode is included in the `ambient` profile. Install Istio with the
following command:
```

The `profiles` support between `istioctl` and `helm` is 1-to-1 - they are the same profiles, in fact.

So either we need to blurb this on the helm page too ("hey, we have other profiles, go look") - or do what that page does, and don't blurb it.

Either way, should be consistent.

Reply:
The sidecar installation docs haven't been meaningfully updated from the time when they were the only installation method.

I agree that a complete overhaul of this is a good idea but I'm not going to boil the ocean in this PR.  I haven't created an issue for this one because I believe that https://github.com/istio/istio/issues/52702 should be general enough for sidecars too.

Reply:
On the topic of ambient specifically, with istioctl you need `profile=ambient` in order to tell it to install ztunnel and istio-cni.  It's reasonable to point that out here - if you only did `istioctl install`, you wouldn't get working ambient mode. 

Given that, in the Helm docs, we explicitly say you need to install Helm charts A B C and D, individually including ztunnel and CNI. What use are profiles there?  Why are these two lines different?

```
$ helm install istiod istio/istiod --namespace istio-system --set profile=ambient --wait
$ helm install ztunnel istio/ztunnel -n istio-system --wait 
```

Reply:
OK, it's explained in https://github.com/istio/istio/blob/master/manifests/helm-profiles/ambient.yaml that they are just sets of values, not groups of components. 

an IstioOperator profile is a list of components and a link to a Helm profile.

this isn't half confusing.

Reply:
Profiles for both are almost entirely (and primarily) sets of values, and in fact both the Istio and helm ones are derived from the same file IIRC.

(they could be both sets of values and of components for both helm and istioctl pretty easily if we did chart composition in helm, but we do not).

Reply:
[The ambient Helm profile](https://github.com/istio/istio/blob/master/manifests/helm-profiles/ambient.yaml) contains environment variables and flags.

[The ambient istioctl profile](https://github.com/istio/istio/blob/master/manifests/profiles/ambient.yaml) contains a list of components to enable, and an instruction to Helm to use the ambient Helm profile.

So yes, you're mostly right, but in the specific case of ambient, the istioctl profile is necessary to say "install ztunnel", where that is spelled out in the Helm docs. 

This has been a useful exercise to understand the distinction, and if we could get chart composition - I assume you get that through [dependencies](https://helm.sh/docs/topics/charts)? - then it would be possible to get the same outcome yes, though not through something called a profile. :)

Reply:
I doubt the helm profiles would work via dependencies unfortunately... helm doesn't have native 'profile' concept so we hack it in a way that would apply the config _after_ it has chosen which dependencies to enable I suspect (untested) (https://blog.howardjohn.info/posts/advanced-helm/)

Reply:
Dependencies in helm can be enabled or disabled based on `values.yaml` properties, so I would _expect_ if you disabled a component but a profile still supplied values for it, that would just no-op in the templating and give you ~roughly the same behavior as `istioctl install <specific component> -f large-operator-yaml-with-many-settings-that-do-not-apply-to-that-component.yaml`

but yeah, need to test.